### PR TITLE
Solve LIBTD-173: Logged in user can change display name in profile by…

### PR DESCRIPTION
**Pull request by tingtingjh**
… making a request with contact form

### CODE REVIEW

File: app/views/users/_profile_actions.html.erb [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
+  <% if current_user == @user %>
+    <%= link_to raw('<i class="glyphicon glyphicon-edit"></i> Edit Your Profile'), sufia.edit_profile_path(@user), class: "btn pull-left subtab" %>
+  <% elsif current_user.following?(@user) %>
+    <%= link_to raw('<i class="glyphicon glyphicon-eye-close"></i> Unfollow'), sufia.unfollow_profile_path(@user), method: :post, class: "btn pull-left subtab" %>
+  <% else %>
+    <%= link_to raw('<i class="glyphicon glyphicon-eye-open"></i> Follow'), sufia.follow_profile_path(@user), method: :post, class: "btn pull-left subtab" %>
+  <% end %>
+  <%= link_to raw('<i class="glyphicon glyphicon-edit"></i> Change Your Display Name'), sufia.contact_path, class: "btn subtab" %>
+  <%= link_to raw('<i class="glyphicon glyphicon-globe"></i> View All Users'), sufia.profiles_path, class: "btn pull-right subtab" %>
+<% end %>
```
Comment (yinlinchen) [inline app/views/users/_profile_actions.html.erb]: Please only submit code changes. If you want to format source code, submit another PR.

**Comment by tingtingjh** [comment]
Reformatting code changes are eliminated.

**Comment by yinlinchen** [comment]
+1
